### ISSUE TITLE
Drag and drop not working to upload attachements using Flatpak

### ISSUE THREAD

**Issue by lguerard** [flathub/org.mozilla.Thunderbird#263]
Hi all,

I recently switched to the Flatpak version of Thunderbird when it got updated to 102. I just realized today that I can't drag and drop files to attach them to emails. 

Is this also happening to other people (aka should I reinstall it) or is this reproduceable ? 

Thanks !

**Comment by TLouf** [comment]
Happening to me too, let me know if you've managed to somehow solve it since.

**Comment by stefan-niedermann** [comment]
I have been managed solving this issue for me by setting `filesystem=home`. It seems like the thunderbird app needs this to be able to access files which are moved into a mail via drag and drop. I used [Flatseal](https://flathub.org/apps/details/com.github.tchx84.Flatseal) for an easy graphical option to change the settings.

**Comment by hzulla** [comment]
I find that a bug. If the user wants to drag and drop something to an application, that's just the same as the user picking that file with a file chooser. Firefox has the same issue, see https://bugzilla.mozilla.org/show_bug.cgi?id=1786251

**Comment by Erick555** [comment]
Yeah, I think fiefox needs to change something in its code to make it work with portals, then thundrbird will use it as well.
